app/templates/ember-users.hbs

```diff
@@ -1,43 +1,41 @@
-<h1>
-  See Who's Using Ember.js
-</h1>
+<section class="container" aria-labelledby="ember-users">
+  <h1 id="ember-users">
+    See Who's Using Ember.js
+  </h1>
 
-<p>
-  Logos are added by company representatives. These companies may or may not be using Ember on their main web properties, but they're definitely using it somewhere in their organizations!
-</p>
-<p>
-  Feel free to track down folks who work at a specific company if you're looking for more detail on their specific usage.
-</p>
+  <p>
+    Logos are added by company representatives. These companies may or may not be using Ember on their main web properties, but they're definitely using it somewhere in their organizations!
+  </p>
+  <p>
+    Feel free to track down folks who work at a specific company if you're looking for more detail on their specific usage.
+  </p>
+
+  <section aria-labelledby="section-ember-users-list-of-companies">
+    <h2 id="section-ember-users-list-of-companies">List of Companies</h2>
 
-<section aria-labelledby="Ember user list">
-  <ul class="user-showcase">
-    {{#each sortedModel as |user|}}
-      <li>
-        <a
-          class="user-showcase__link"
-          href={{user.url}}
-          rel="nofollow noopener"
-          target="_blank"
+    <ul class="list-unstyled layout-grid">
+      {{#each this.sortedModel as |user|}}
+        <EsCard
+          class="col-1-large"
+          @image={{asset-map (concat "images/users/" user.image)}}
+          @alt={{user.name}}
```

Comment:
alt text should not be repeated as text - since the company name is here, the alt text should be blank and `role="presentation"` should be added.

Reply:
Ah, ok. That makes sense. Will make that change soon.

On a related note, can I show the company's name in a card? The current Guides only shows the company's logo, so I wasn't sure of this change.

Reply:
Note: Adding the presentation role will be completed in https://github.com/ember-learn/ember-styleguide/pull/243

Reply:
> On a related note, can I show the company's name in a card? The current Guides only shows the company's logo, so I wasn't sure of this change.

I kind of like it, although I think we'd need a little more styling to make it work.

Reply:
@ijlee2 for now we should go with your solution of using the yield to wrap the image only with a link instead of adding the company name.

This sort of change is not something that we can decide like this unfortunately and will require a large amount of data wrangling that is not in the scope of this issue/PR

Reply:
@mansona Ok, sounds good. I'll try out `yield`ing when I get a chance today.

Reply:
@mansona @MelSumner While yielding `<a>` and `<img>` worked and matched better the current webpage, this presented a few problems in return.

I think these problems are minor (relative to my guess for this webpage's importance/current traffic) and could be addressed in a future ticket. What do you think?

- For a yielded `<a>` (without `h3` surrounding it), not *all* of the styles for `card-link` apply. I can see the background color change on hover, but I'm not able to click anywhere on the card to open the link. (I have to click on the company image.) This is because the pseudoelement styles that allow the card to be clicked anywhere are defined in selectors `h3 a::before` and `h3 a::after`.
- A yielded `<img>` doesn't take full width, even with the parameter `full-image` on `<EsCard>`. Currently, that parameter and its corresponding CSS styling apply if we use `@image` attribute.
- When tabbing through the list, the blue box's position and dimension don't look great. It looks like the existing `outline` style doesn't apply well for a yielded `<a>` tag.
- I still haven't solved the problem of showing fewer cards across width for smaller resolutions.

<img width="1680" alt="Screen Shot 2019-11-02 at 9 07 02 AM" src="https://user-images.githubusercontent.com/16869656/68072206-0a63c600-fd51-11e9-8611-f90f93ffef29.png">